google/api_core/operations_v1/__init__.py

```diff
@@ -14,6 +14,12 @@
 
 """Package for interacting with the google.longrunning.operations meta-API."""
 
+import sys
+
 from google.api_core.operations_v1.operations_client import OperationsClient
 
-__all__ = ["OperationsClient"]
+if sys.version_info[0] >= 3 and sys.version_info[1] >= 6:
+    from google.api_core.operations_v1.operations_async_client import OperationsAsyncClient  # noqa: F401
+    __all__ = ["OperationsClient", "OperationsAsyncClient"]
+else:
+    __all__ = ["OperationsClient"]  # pragma: NO COVER
```

Comment:
In other python code I've seen the convention be a little more like
```python
__all__ = ["OperationsClient"]
if sys.version_info >= (3, 6, 0):
    from google.api_core.operations_v1.operations_async_client import OperationsAsyncClient  # noqa: F401
    __all__.append("OperationsAsyncClient")
```
It makes coverage a little easier in this case, too.

Reply:
Good catch. The coverage ignore comment is removed.